wordpress/wp-content/plugins/sil-dictionary-webonary/webonary/Webonary_Cloud.php

```diff
@@ -329,4 +329,55 @@ public static function searchEntries($posts, WP_Query $query) {
 
 		return null;
 	}
+
+	public static function registerApiRoutes()
+	{
+		$namespace = 'webonary-cloud/v1';
+
+		register_rest_route($namespace, '/validate', array(
+				'methods' => 'POST' | WP_REST_Server::CREATABLE,
```

Comment:
I'm not sure what this bitwise operator (` | `) is trying to accomplish. If both values are `POST` the result will be `POST`. Otherwise the result will be a mangled string. For example `POST | GET` returns `WOWT`.

Also, `POST | WP_REST_Server::ALLMETHODS` returns `WOW| POST, PUT, PATCH, DELETE`

Reply:
Yikes... Thanks for pointing this out. I was not too familiar with what this function was doing, so I copied legacy code from WEBONARY_API_MyAPI.php and changed it for my purposes. I see from the doc at https://developer.wordpress.org/reference/classes/wp_rest_server/ that there is no need for this bitwise operator at all, since `const CREATABLE = 'POST';` and `const READABLE = 'GET';`. So I am modifying the code to just use these constants. Thanks again for our vigilence.